jetty-maven-plugin/src/main/java/org/eclipse/jetty/maven/plugin/JettyStopMojo.java

```diff
@@ -220,9 +220,19 @@ private String send(String command, int wait)
             }
             else
             {
-                //Wait only a small amount of time to ensure TCP has sent the message
-                s.setSoTimeout(1000);
-                s.getInputStream().read();
+                try
+                {
+                    //Wait only a small amount of time to ensure TCP has sent the message
+                    s.setSoTimeout(1000);
+                    s.getInputStream().read();
+                }
+                catch (Exception e)
+                {
+                    if (getLog().isDebugEnabled())
+                        getLog().error("Error after sending command: " + command + ". Check the server state.", e);
+                    else
+                        getLog().info("Error after sending command: " + command + ". Check the server state.");
```

Comment:
Maybe "Error" could be `e.getMessage()`? It's probably `SocketException: Connection reset` but could be something else.

Reply:
OK, how about the latest commit then?

Reply:
Looks great! It's too bad we have to work around Windows TCP issues like this but it will be a quality of life improvement for me and others. Thanks!